travel/src/test/java/com/tx/travel/service/ExchangeRateServiceLocalImplTest.java

```diff
@@ -0,0 +1,288 @@
+package com.tx.travel.service;
+
+import static org.junit.jupiter.api.Assertions.*;
+import static org.mockito.Mockito.*;
```

Comment:
avoid using star imports, only use specific classes

Reply:
Noted! Thank you